doc/buildAndProgram.md

```diff
@@ -85,9 +86,19 @@ If you just want to build the project and run it on the Pinetime, using *pinetim
 Build:
 
 ```
+cd build
 make -j4 pinetime-app
 ```
 
+However if you want to flash the firmware OTA using an app like Gadgetbridge, using *pinetime-mcuboot-app* will create the `pinetime-mcuboot-app-dfu-$VERSION.zip` file you want in the `build/src` folder.
```

Comment:
This is a great addition. Maybe modify the above L84 to specify "PineTime devkit"? Currently these instructions kind of assume a devkit, whereas I think a lot of people work on sealed now as the bootloader is very stable